### ISSUE TITLE
Regarding VNM, it caused the crash to be BUG! ! Not Voidwatch NM

### ISSUE THREAD

**Issue by Freetate** [DerpyProjectGroup/topaz#1493]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened
- [x] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I have specified what branch this happens on branch: EDITME

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 
[VNM  BUG.txt](https://github.com/LandSandBoat/server/files/8405060/VNM.BUG.txt)

Operating procedures:
Player A (any class will do) generates VNM.
Player B (must be SMN)

Player A is killed by VNM
When player A dies, VNM is invisible for an instant

Player B enters the command for pet to attack VNM
Regardless of whether the VNM is attacked, as long as the command is output
will cause a crash

**Comment by zach2good** [comment]
> Exception code: 80000003 (BREAKPOINT)

Are you running in debug mode?

``` 
[04/04/22 00:14:30:242][critical][stacktrace] === Call stack ===
[04/04/22 00:14:30:242][critical][stacktrace] Address           Frame             Function
[04/04/22 00:14:30:437][critical][stacktrace] 00007FF7A6F37627  000000C164CFBE70  CLuaBaseEntity::messageSpecial+187 (C:\server\src\map\lua\lua_baseentity.cpp, line 429)
[04/04/22 00:14:30:466][critical][stacktrace]     Local CLuaBaseEntity* this = 0x2907F8BA620
```

This is the problem line:

https://github.com/LandSandBoat/server/blob/base/src/map/lua/lua_baseentity.cpp#L429

**Comment by zach2good** [comment]
Probably a side effect of this:
https://github.com/LandSandBoat/server/blob/base/scripts/globals/voidwalker.lua#L359-L363

`target` should always be a char

**Comment by TeoTwawki** [comment]
yeah that needs to check if the target is a player and exists. mob could lose target or change target to a pet before message prints. Or more accurately...should not even be happening in onMobFight. Just like these mobs should not be up in the first place. The whole invisible untargettable mob thing is wrong these need to be non spawned till the player rests in the right spot.
